src/spec-node/dockerCompose.ts

```diff
@@ -688,3 +688,38 @@ export function dockerComposeCLIConfig(params: Omit<PartialExecParameters, 'cmd'
 		})());
 	};
 }
+
+/**
+ * Convert mount command' arguments to Docker Compose volume
```

Comment:
nit: extra '

Reply:
Fixed in d2510412db8afb196ab6e8f852021de2338d8c30